src/website/shared/models/timestamps.py

```diff
@@ -0,0 +1,25 @@
+"""Timestamp mixins
```

Comment:
Arguably, it seems overkill to me to collect a file only of timestamp mixins, I'd recommend renaming this to mixins and making this a file of all models mixins we care about.

Reply:
I can rename the file for this PR. However I won't move other mixins here in this PR as that would increase even more the testing surface due to the general lack of automated tests.

The whole purpose of moving the timestamps together was to increase the visibility of the comments that hints at strategies to manage timestamps in Django according to the desired effect.